config/kernel-vfs-iov_iter.m4

```diff
@@ -122,10 +133,17 @@ AC_DEFUN([ZFS_AC_KERNEL_VFS_IOV_ITER], [
 		AC_MSG_RESULT(yes)
 		AC_DEFINE(HAVE_IOV_ITER_FAULT_IN_READABLE, 1,
 		    [iov_iter_fault_in_readable() is available])
+	],[
+	AC_MSG_CHECKING([whether fault_in_iov_iter_readable() is available])
+	ZFS_LINUX_TEST_RESULT([fault_in_iov_iter_readable], [
+		AC_MSG_RESULT(yes)
+		AC_DEFINE(HAVE_FAULT_IN_IOV_ITER_READABLE, 1,
+		    [fault_in_iov_iter_readable() is available])
```

Comment:
nit: It looks like the indentation isn't quite right here.

Reply:
And fixed.